src/main/java/Duke.java

```diff
@@ -1,10 +1,198 @@
+import java.util.ArrayList;
+import java.io.*;
+import java.util.Scanner;
+
 public class Duke {
+
+    static int idx = 0;
+    static ArrayList<Task> tasks = new ArrayList<>();
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+        greetUser();
+
+        loadData();
+
+        serveUser();
+
+        goodbyeUser();
+
+    }
+
+    private static void loadData() {
+        try {
+            BufferedReader in = new BufferedReader(new FileReader("Duke.txt"));
+            String line = in.readLine();
+            String isDoneStatus, command, input;
+            String[] temp;
+            while (line != null) {
+                temp = line.split(" ", 2);
+                isDoneStatus = temp[0];
+                input = temp[1];
+                temp = temp[1].split(" ", 2);
+                command = temp[0].toLowerCase();
+                switch (command) {
```

Comment:
I think the indentation of the switch statements is wrong. Please refer to the coding standards